python-modules/chevah-python-test/test_python_binary_dist.py

```diff
@@ -298,17 +298,40 @@ def main():
         sys.stderr.write('standard "ssl" missing.\n')
         exit_code = 2
 
-    # cryptography module and pyOpenSSL 16.0.0 are only available on ArchLinux
-    if chevah_os == "archlinux":
+    # cryptography module and latest pyOpenSSL are only available on
+    # few systems.
+    if chevah_os in [
+        'archlinux',
+        'openbsd58',
+        'osx108',
+        'raspbian7',
+        'rhel5',
+        'rhel6',
+        'rhel7',
+        'sles11',
+        'sles12',
+        'solaris11',
+        'ubuntu1204',
+        'ubuntu1404',
+        'ubuntu1604',
+        'windows',
+            ]:
```

Comment:
Can't we use the `$BUILD_CFFI` environment variable here to avoid duplicating the `if` conditions in both shell and Python code?